packages/e2e-tests/specs/editor/blocks/site-title.test.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	createUser,
+	deleteUser,
+	insertBlock,
+	loginUser,
+	pressKeyWithModifier,
+	switchUserToAdmin,
+	switchUserToTest,
+	visitAdminPage,
+} from '@wordpress/e2e-test-utils';
+
+async function getOption( setting ) {
+	await switchUserToAdmin();
+	await visitAdminPage( 'options.php' );
+
+	const value = await page.$eval(
+		`#${ setting }`,
+		( element ) => element.value
+	);
+	await switchUserToTest();
+	return value;
+}
+
+async function setOption( setting, value ) {
+	await switchUserToAdmin();
+	await visitAdminPage( 'options-general.php' );
+
+	await page.focus( `#${ setting }` );
+	await pressKeyWithModifier( 'primary', 'a' );
+	await page.type( `#${ setting }`, value );
+
+	await Promise.all( [
+		page.click( '#submit' ),
+		page.waitForNavigation( { waitUntil: 'networkidle0' } ),
+	] );
+
+	await switchUserToTest();
+}
+
+const saveEntities = async () => {
```

Comment:
We're starting to have these multi-entities saving steps in a couple of tests (multi-entities, reusable-blocks...), it looks like we'd probably need to factorize these in some utility.